plugin/utils/flag.py

```diff
@@ -275,6 +275,7 @@ def build(self):
         "-mthread-model",
         "-o",
         "-serialize-diagnostics",
+        "--serialize-diagnostics",
```

Comment:
We either would have to add all with `--` or none. I am not a fan of having one flag different from other.

Reply:
There is not much flags that support both prefixes. I have no idea how to do it better way.